infra/terraform/modules/cloudwatch_alerts/cloudwatch.tf

```
@@ -47,3 +47,28 @@ resource "aws_cloudwatch_metric_alarm" "cpu_low_threshold" {
 
   tags = "${var.tags}"
 }
+
+# alert if an EC2 instance becomes unhealthy for more than 1 period of time
+resource "aws_cloudwatch_metric_alarm" "health_monitoring" {
+  count             = "${length(var.ec2_instance_names)}"
+  alarm_name        = "${var.name}_${element(var.ec2_instance_names, count.index)}-unhealthy"
+  alarm_description = "This metric monitors EC2 health status"
+
+  dimensions = {
+    InstanceId = "${element(var.ec2_instance_ids, count.index)}"
+  }
+
+  namespace           = "AWS/EC2"
+  metric_name         = "StatusCheckFailed"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  threshold           = "2"
+  period              = "${var.period}"
+  evaluation_periods  = "${var.evaluation_periods}"
```

As per what @andylightfoot said above, I think this needs to be 

```suggestion
  evaluation_periods  = "${var.period}"
```

---

This one looks OK to me:

  evaluation_periods:                    "3"
      metric_name:                           "StatusCheckFailed"
      namespace:                             "AWS/EC2"
      period:                                "300"